tools/mksurfdata_esmf/gen_mksurfdata_jobscript_single.py

```diff
@@ -43,6 +43,14 @@ def get_parser():
         dest="tasks_per_node",
         required=True,
     )
+    parser.add_argument(
+        "--machine",
+        help="""currently this recognizes cheyenne and casper (default cheyenne)""",
```

Comment:
Add a note to this that this needs to be the cime machine. So it's a machine that has been ported to cime and you can use it to build a cime model. You might also show the link to list cime machines and how to port a cime model.